awx/main/scheduler/task_manager.py

```diff
@@ -572,10 +555,19 @@ def start_task(self, task, instance_group, dependent_tasks=None, instance=None):
             task.save()
             # TODO: run error handler to fail sub-tasks and send notifications
         else:
+            if type(task) is WorkflowJob:
+                task.status = 'running'
+                task.send_notification_templates('running')
+                logger.debug('Transitioning %s to running status.', task.log_format)
+                # Call this to ensure Workflow nodes get spawned in timely manner
+                ScheduleWorkflowManager().schedule()
             # at this point we already have control/execution nodes selected for the following cases
-            task.instance_group = instance_group
-            execution_node_msg = f' and execution node {task.execution_node}' if task.execution_node else ''
-            logger.debug(f'Submitting job {task.log_format} controlled by {task.controller_node} to instance group {instance_group.name}{execution_node_msg}.')
+            else:
+                task.instance_group = instance_group
+                execution_node_msg = f' and execution node {task.execution_node}' if task.execution_node else ''
+                logger.debug(
+                    f'Submitting job {task.log_format} controlled by {task.controller_node} to instance group {instance_group.name}{execution_node_msg}.'
+                )
```

Comment:
Just marking for notes - this seems to bring us back to parity with `devel`.

The decisions here have significant system consequences and are non-obvious, so this is best. For example, from the cancel rework stuff, I know that workflows are assigned a `celery_task_id`. Because of that, I have to special-case workflows to not look for the process running under the dispatcher service. Nonetheless, that's accepted canonical behavior, so we shouldn't change it.